logaggregator/api.go

```diff
@@ -73,24 +124,27 @@ func NewMessageFromSyslog(m *rfc5424.Message) client.Message {
 	}
 }
 
-// TODO(bgentry): does this belong in the syslog package?
-func splitProcID(procID []byte) (processType, jobID string) {
-	split := bytes.Split(procID, []byte{'.'})
+var procIDsep = []byte{'.'}
+
+func splitProcID(procID []byte) (processType, jobID []byte) {
+	split := bytes.Split(procID, procIDsep)
 	if len(split) > 0 {
-		processType = string(split[0])
+		processType = split[0]
 	}
 	if len(split) > 1 {
-		jobID = string(split[1])
+		jobID = split[1]
 	}
 	return
 }
 
 func streamFromMessage(m *rfc5424.Message) string {
-	switch m.Severity {
-	case 3:
-		return "stderr"
-	case 6:
+	switch string(m.MsgID) {
+	case "ID1":
 		return "stdout"
+	case "ID2":
+		return "stderr"
+	case "ID3":
+		return "initLog"
```

Comment:
We are not supposed to be doing that, no. Looks like I missed deleting this line when I removed the special handling of initLog stuff yesterday. Will nuke it :v: